charts/opentelemetry-kube-stack/templates/collector.yaml

```diff
@@ -32,6 +32,8 @@ spec:
   image: "{{ $collector.image.repository }}@{{ $collector.image.digest }}"
   {{- else if $collector.image.tag }}
   image: "{{ $collector.image.repository }}:{{ $collector.image.tag }}"
+  {{- else if $collector.image.repository }}
+  image: "{{ $collector.image.repository }}:{{ $.Chart.AppVersion }}"
```

Comment:
I think we could improve this by doing what the collector chart does `{{ .Values.image.tag | default .Chart.AppVersion }}`